debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java

```diff
@@ -407,8 +407,7 @@ protected static Serializable deserializeDatetimeV2(int meta, ByteArrayInputStre
     protected static Serializable deserializeTimestamp(ByteArrayInputStream inputStream) throws IOException {
         long epochSecond = inputStream.readLong(4);
         int nanoSeconds = 0; // no fractional seconds
-        LocalDateTime localDateTime = LocalDateTime.ofEpochSecond(epochSecond, nanoSeconds, ZoneOffset.UTC);
-        return ZonedDateTime.of(localDateTime, ZoneId.systemDefault());
+        return ZonedDateTime.ofInstant(Instant.ofEpochSecond(epochSecond, nanoSeconds), ZoneId.systemDefault());
```

Comment:
I'm having some doubts about the usefulness of using the server timezone for "rendering" the timestamp.

I think it'd have been better to emit this as UTC, or perhaps a TZ configurable via a connector option. But no need to change for now, let's see if someone asks for it. And one can always restore UTC and convert to whatever TZ needed as the offset is part of the conveyed String.